src/components/MovieDetails/index.tsx

```diff
@@ -86,7 +86,7 @@ const MovieDetails: React.FC<MovieDetailsProps> = ({ movie }) => {
   const intl = useIntl();
   const { locale } = useLocale();
   const [showManager, setShowManager] = useState(
-    router.query.manage !== undefined ? true : false
+    router.query.manage !== '1' ? true : false
```

Comment:
I'm confused. If it _is_ 1, we should set this to true?

Reply:
Uh, yeah, whoops.  🤦🏻‍♀️